src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -1434,7 +1434,12 @@ MPU9250::measure()
 	arb.scaling = _accel_range_scale;
 	arb.range_m_s2 = _accel_range_m_s2;
 
-	_last_temperature = (report.temp) / 361.0f + 35.0f;
+	// We calculate sensor temperature as :
+	// temperature_real = temperature_raw * temperature_sensitivity + zero_offset
+	//
+	// Sensitivity (in degC/LSB) is from the Invensense datasheet for the specific sensor
+	// Zero offset (in degC) varies from part to part, and this value is a good default
+	_last_temperature = (report.temp) / 340.0f + 21.0f;
```

Comment:
Where do these numbers come from? Is the temp sensitivity not 333.87LSB/degC?
![mpu9250_temp_sensitivity](https://user-images.githubusercontent.com/4036885/37334885-6962a396-26ad-11e8-8a1e-9a4c6e7750bf.png)

Reply:
Yep, you're right. Thanks!